contracts/GateSeal.vy

```diff
@@ -167,13 +220,19 @@ def seal(_sealables: DynArray[address, MAX_SEALABLES]):
         # for details, see https://docs.vyperlang.org/en/stable/built-in-functions.html#raw_call
         success, response = raw_call(
             sealable,
-            _abi_encode(SEAL_DURATION_SECONDS, method_id=method_id("pauseFor(uint256)")),
+            abi_encode(SEAL_DURATION_SECONDS, method_id=method_id("pauseFor(uint256)")),
             max_outsize=32,
```

Comment:
`max_outsize=32`

this was only needed to bypass the bug. Now that it's fixed and we don't actually need the return data, we can set it to `0`.